cmd/scollector/collectors/redis_linux.go

```diff
@@ -1,5 +1,3 @@
-// +build darwin linux
```

Comment:
Because part of this collector is executing `ps`, which will likely fail on windows.